plenum/persistence/rocksdb_hash_store.py

```diff
@@ -0,0 +1,85 @@
+from ledger.hash_stores.hash_store import HashStore
+from storage.kv_store_rocksdb import KeyValueStorageRocksdb
+from stp_core.common.log import getlogger
+
+
+logger = getlogger()
+
+
+class RocksDbHashStore(HashStore):
```

Comment:
The implementation looks exactly the same as `LevelDbHashStore`. Maybe use just one implementation with Dependency Injection of a proper key-value implementation?

Reply:
Yes, but I propose to merge hash stores implementations after rocksdb build procedure is ready.

Reply:
Done.